catalog/spatial/geocoding/spatial-geocoding-websearch/pom.xml

```diff
@@ -62,7 +64,9 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>json-smart,spatial4j,spatial-geocoding-api-impl</Embed-Dependency>
+                        <Embed-Dependency>
+                            json-smart,spatial4j,spatial-geocoding-api-impl,commons-lang3
```

Comment:
I don't recall the code formatter having a preference for comma-delimited lists within xml files - it would reformat a _lot_ of files that way - but even if it does, you can always manually restore that one block. You could even add formatter:off/on directives to the file but that's a bit of overkill.

Finally, if the formatter _is_ doing that, it should probably be updated so that it does not.